AngularBFF/AngularBFF.Net8.Api/Program.cs

```diff
@@ -0,0 +1,58 @@
+using AngularBFF.Net8.Api.Http;
+using AngularBFF.Net8.Api.Weather;
+using Fhi.HelseId.Web.ExtensionMethods;
+
+var builder = WebApplication.CreateBuilder(args);
+
+if (builder.Environment.IsDevelopment())
+{
+    builder.Configuration.AddUserSecrets("b1a93959-6172-416e-bd25-8d43347eb8f3");
+}
+
+builder.Configuration
+.AddJsonFile("appsettings.json", optional: false, reloadOnChange: true)
+.AddJsonFile($"appsettings.{builder.Environment.EnvironmentName}.json", optional: true)
```

Comment:
If the ASPNETCORE_ENVIRONMENT environment variable is set, we do not need to set this. I also think we get many of the other options out of the box, have a look here: https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-9.0#default-application-configuration-sources

Reply:
Not my code (I only renamed) ;). But you are correct. I will cleanup.